lib/ms2ex/handlers/login/response_key.ex

```diff
@@ -22,6 +22,10 @@ defmodule Ms2ex.LoginHandlers.ResponseKey do
     {token_a, packet} = get_int(packet)
     {token_b, _packet} = get_int(packet)
 
+    IO.inspect(token_a, label: "A")
+    IO.inspect(token_b, label: "B")
+    IO.inspect(auth_data)
```

Comment:
if this is relevant data, should be a Logger rather than a inspect

Reply:
Nah, i forgot to remove them.